### ISSUE TITLE
Bug Report (Although installed, "Cascadia Code" doesn't show up inside Apache Netbeans)

### ISSUE THREAD

**Issue by 1chriis1** [microsoft/cascadia-code#238]
# Environment

```
Cascadia Code version number: 1911.21
Application (with version) used to display text: Netbeans 8.1 and Netbeans 11.1
OS platform and version: Windows 10 Version 10.0.18363 Build 18363

Screen resolution (i.e. 220dpi): dunno

Any other software?
```

# Steps to reproduce

Open Netbeans, go to Tools, Options, Fonts, and open the font picker. 
Cascadia Code isn't there at all.

# Expected behavior

It should be in the list of fonts in my system, in all apps that support changing the font.

# Actual behavior

Cascadia code doesn't show up when trying to pick a default font in Apache Netbeans.

**Comment by cDurham** [comment]
Also having this issue

**Comment by 1chriis1** [comment]
😂 Thought this was quite a niche problem to have, but apparently more people are using Netbeans and want a clean and nice font.

**Comment by aaronbell** [comment]
Can you share an example of a downloaded font that does work in Netbeans? Thanks!

**Comment by 1chriis1** [comment]
@aaronbell
Fira Mono(strangely not Fira Code), Everson, iA Writer Duospace, IBM Plex (curiously all variants of it), Inconsolata, Product Sans, Roboto, SF mono(not the Apple original one, [this one](https://github.com/ZulwiyozaPutra/SF-Mono-Font) ) and Space Mono.

Those are the ones that I could catch with a quick run through my fonts on Netbeans.

It's not a big bummer to not have Cascadia in NB, but it's quite strange that it doesn't appear.
I even tried and deleted and re-installed it. Same results.

**Comment by gkuhlmann** [comment]
I had the same issue with the latest stable NetBeans version (11.3) running on JDK11 on Windows10

An [old StackOverflow question](https://stackoverflow.com/a/55752283) pointed me to a solution that worked for me:

> One some Windows machines there are two buttons for installing fonts: Install For Me and Install For All Users. Java only lists fonts installed for all users.

**Comment by aaronbell** [comment]
I investigated this issue and am not able to repro the bug:

<img width="595" alt="Capture4" src="https://user-images.githubusercontent.com/8460297/80562259-09c48900-899c-11ea-8e76-4ff8dcd97a12.PNG">

My settings:
Windows 10 Pro
Apache NetBeans IDE 11.3
Java Development Kit 14.0.1

In looking around online, I found [this article](https://stackoverflow.com/questions/55112673/newly-installed-fonts-do-not-appear-in-netbeans) which might help you. 

As this does not appear to be a font problem, I am closing this issue. Thanks!
